package.json

```
@@ -335,14 +345,16 @@
 					},
 					"additionalProperties": false
 				},
-				"cursorless.maximumHatStylePenalty": {
+				"cursorless.hatSettings.hatPenalties.maximumHatStylePenalty": {
```

This doesn't really have any impact on hat penalties, so I don't think I'd put it in that group
```suggestion
				"cursorless.hatSettings.maximumHatStylePenalty": {
```